ui/src/components/VideoOverlay.tsx

```diff
@@ -361,18 +361,18 @@ export function NoAutoplayPermissionsOverlay({
 }
 
 interface PointerLockBarProps {
-  show: boolean;
+  readonly show: boolean;
 }
 
 export function PointerLockBar({ show }: PointerLockBarProps) {
   return (
     <AnimatePresence mode="wait">
       {show ? (
         <motion.div
-          className="absolute -top-[36px] left-0 right-0 z-20 bg-white"
-          initial={{ y: 20, opacity: 0, zIndex: 0 }}
-          animate={{ opacity: 1, y: 0 }}
-          exit={{ y: 43, zIndex: 0 }}
+          className="flex w-full items-center justify-between bg-transparent"
```

Comment:
Animating the location just plain broke the layout and resulted in this button/action just being behind and invisible to the user. I couldn't get the thing centered the way I wanted, but in this form it animates into and out of visibility correctly. Also, setting the `motion.div` to`bg-transparent` helps ensure it doesn't look ugly.